### ISSUE TITLE
No 'latest' release (but 'latest' tag present)

### ISSUE THREAD

**Issue by tenzap** [validator/validator#2092]
### Summary

There is no 'latest' release at the moment, but the 'latest' tag is present.

Could it be that the release was deleted but not recreated because the latest CI run failed?

### URL for document that reproduces the problem

n/a

### HTML/CSS/SVG snippet that reproduces the problem

```html
n/a
```

### Checker error messages

```shell
n/a
```

### Steps to reproduce

1. n/a
2.
3.

### Expected behavior

n/a

### Actual behavior

n/a

**Comment by phpfui** [comment]
I just noticed this as well. The URL https://github.com/validator/validator/releases/download/latest/vnu.jar has worked in the past, but now returns a 404 status.

Not sure if this a github issue, or a repo issue.

**Comment by sideshowbarker** [comment]
The cause here is the GitHub regularly puts the `latest` release into the Draft state, and the only way I've been able to find to fix that is to manually push the Publish button to make non-draft.

But it looks like I'm need to set up and other kind of workaround — for example, some cron job which checks the state and changes it to Publish if it's in Draft (that is, the check I otherwise need to do manually).

**Comment by phpfui** [comment]
Thanks for the quick update. I am wondering if a github action could do this job, then it would be automatic. I don't deploy built executables, but I also publish full releases, and not just tags. Maybe a full Release process might be a better workflow that would do this more naturally.

Thanks for maintaining this project. I use it to validate all my HTML so my sites return 100% valid HTML and CSS!
